README.md

```diff
@@ -1 +1,34 @@
 # trails-review
+
+prepare portfolio website
```

Comment:
please add a screenshot from the life demo

Reply:
So It can be more professional

Reply:
- If you need to edit a comment, you could click on the 3 dots and click edit, this is better than create another comment. 🐶
![image](https://user-images.githubusercontent.com/20563252/130513589-9d8dacf5-8fa6-4e21-9eef-4d5ab551262d.png)